optimum/onnx/configuration.py

```diff
@@ -340,6 +340,61 @@ def fill_with_past_key_values_(self, inputs_or_outputs: Mapping[str, Mapping[int
             inputs_or_outputs[f"{name}_key_values_{i}"] = {0: "batch", 2: decoder_sequence}
 
 
+class DecoderOnnxConfigWithPast(OnnxConfigWithPast):
```

Comment:
Would it be possible to make this compatible (and use) `optimum.exporters.onnx`? This can be done in later PRs but just asking!

Reply:
Yes that would be the plan ! And it makes sense to add it in an other PR